### PR TITLE
frozen-bubble: Add appstream metadata

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Added appstream metadata (Part of getsolus/packages#1389)
- Added StartupWMClass to desktop file

**Test Plan**

Verify metadata with `appstream-builder --packages-dir=. --include-failed -v`

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/frozen-bubble/files/frozen-bubble.desktop [hunk 1]
```diff
@@ -4,5 +4,6 @@ Comment=Frozen Bubble arcade game
 Exec=frozen-bubble
 Icon=frozen-bubble
 Terminal=false
+StartupWMClass=perl
```
Comment (EbonJaeger) [inline packages/f/frozen-bubble/files/frozen-bubble.desktop:7]: Out of curiosity, why is the startup class 'perl'?

Comment (malfisya) [reply packages/f/frozen-bubble/files/frozen-bubble.desktop:7]: Screenshot from 2024-01-26 05-15-51

I don't like it, but it works. Is there any tool on Budgie like this(Looking Glass)? to check wmclass?

Comment (EbonJaeger) [reply packages/f/frozen-bubble/files/frozen-bubble.desktop:7]: You should be able to run `xprop WM_CLASS` in a terminal, and then click on the window you want to inspect.

Comment (malfisya) [reply packages/f/frozen-bubble/files/frozen-bubble.desktop:7]: it is the same after all in Plasma and Budgie. I hope this is a non-issue

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! :)

**Pull request merged**